### DISCUSSION TITLE
Can't see my server in the jazz directory

### DISCUSSION THREAD

**Discussion by openaspace** [jamulussoftware/jamulus#2981] [Technical Support & Help]
I can't see my server registere in the jazz directory...
.. no errors in the log.. the server works from wan ip and lan ip...udp port opened..

```
[Unit]
Description=Jamulus headless server
After=network.target
StartLimitIntervalSec=0

[Service]
Type=simple
User=jamulus
Group=nogroup
NoNewPrivileges=true
ProtectSystem=true
ProtectHome=true
Nice=-20
IOSchedulingClass=realtime
IOSchedulingPriority=0

ExecStart=/bin/sh -c 'exec /usr/bin/jamulus-headless -s -n'
--fastupdate \
--serverinfo "ExperimentalImpro;Bologna;106" \
--directoryserver jazz.jamulus.io:22324 \
--welcomemessage "<h4>Only Free Experimental Improvisations</h4> \
#--htmlstatus "/var/www/html/JamulusServerStatus.dat" \
--numchannels 20 \
--multithreading \
--log "/var/log/jamulus/jamulus.log" \


Restart=on-failure
RestartSec=30
StandardOutput=journal
StandardError=inherit
SyslogIdentifier=jamulus


[Install]
WantedBy=multi-user.target
```

**Comment by rdica** [comment]
This one as shown on https://explorer.jamulus.io ?

36 | Experimental-Impro | Bologna | Italy | 0 / 10 | 164 | xx.xx.xxx.xx | 22124 | 3.9.1 | Linux

**Reply by openaspace** [reply]
Yes solved

**Reply by openaspace** [reply]
Remove the ip from your comment. thanx.

> This one as shown on https://explorer.jamulus.io ?
>

**Reply by ann0see** [reply]
@openaspace I’ve removed it but keep in mind that you can always get the IP as long as you have access to Jamulus explorer or connect as client.

**Reply by openaspace** [reply]
I know all ;)

**Comment by pljones** [comment]
For future reference, the output of `systemctl status <servicename>` and `journalctl -u <servicename> --since <starttime>` are very useful.

**Reply by openaspace** [reply]
it didn't report any errors the problem was in the startup options declaration syntax between the maybe old mode with / and the new one in the exec between ' '

**Reply by pljones** [reply]
The output is still useful, regardless, when reporting an issue.
